group2_assignment2_solution.ipynb

```diff
@@ -736,18 +759,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 23,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "TypeError",
+     "evalue": "unsupported operand type(s) for -: 'list' and 'list'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
+      "\u001b[0;32m<ipython-input-23-0164fb9fd226>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0;31m# INSERT YOUR CODE HERE\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 8\u001b[0;31m \u001b[0mk1_removed\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mk1_tweets\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0mk1_tweets_filtered\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      9\u001b[0m \u001b[0mk2_removed\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mk2_tweets\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0mk2_tweets_filtered\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     10\u001b[0m \u001b[0mk3_removed\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mk3_tweets\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0mk3_tweets_filtered\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mTypeError\u001b[0m: unsupported operand type(s) for -: 'list' and 'list'"
+     ]
+    }
+   ],
    "source": [
     "# YOU ARE REQUIRED TO INSERT YOUR CODES IN THIS CELL\n",
     "'''\n",
     "For each keyword, print out the number of tweets that have been removed.\n",
     "'''\n",
     "\n",
-    "# INSERT YOUR CODE HERE"
+    "# INSERT YOUR CODE HERE\n",
+    "\n",
+    "k1_removed = len(k1_tweets) - k1_tweets_filtered\n",
+    "k2_removed = k2_tweets - k2_tweets_filtered\n",
+    "k3_removed = k3_tweets - k3_tweets_filtered\n",
```

Comment:
I haven't tried this in a cell, but should these be something like: 
`k1_removed = len(k1_tweets - k1_tweets_filtered)`?